pkg/plugins/policies/meshaccesslog/plugin/v1alpha1/plugin_test.go

```diff
@@ -25,6 +26,11 @@ import (
 )
 
 var _ = Describe("MeshAccessLog", func() {
+
+	core.TempDir = func() string {
+		return "/tmp"
+	}
```

Comment:
Don't you need to restore the TempDir?

Reply:
Each spec runs as a separate process, so we don't have to restore the TempDir, but this code should be inside `BeforeEach`:
```go
func BeforeEach() {
	core.TempDir = func() string {
		return "/tmp"
	}
}
```
I'll fix this in other places as well

Reply:
Fixed